tests/GraphQLTest.php

```diff
@@ -215,10 +216,6 @@ public function testFormatValidationError()
      */
     public function testAddType()
     {
-        $this->expectsEvents(TypeAdded::class);
-	
-	    $this->app['events']->shouldReceive('listen');
```

Comment:
IMHO this isn't correct and pointed out in the config => the default should still be true.